recipes/cbgen/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "cbgen" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4a881d6cf0a73a53fdae82088fbf4777e6c32fcf352a9f482038d36de245e7e3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - bgen >=4.1.0
+    - cffi >=1.14.0
+    - pip
+    - pytest-runner >=5.2
+    - python
+    - setuptools
+    - setuptools-scm >=4.1.2
+    - wheel >=0.34.2
+  run:
+    - appdirs >=1.4.3
+    - bgen >=4.1.0
+    - cffi >=1.14.0
+    - numpy >=1.17.0
+    - pooch >=1.1.1
+    - pytest >=5.4.1
```

Comment:
Is pytest really needed at run time?

Reply:
Yes because I allow the user to test the package at runtime:

```python
import cbgen

cbgen.test()
```

Reference: https://github.com/limix/cbgen/blob/master/cbgen/_testit.py

Reply:
You can use the `tests_require` entry for that.